gradle-plugins/compose/src/test/test-projects/application/macOptions/build.gradle

```diff
@@ -28,12 +28,36 @@ compose.desktop {
         mainClass = "MainKt"
         nativeDistributions {
             packageName = "TestPackage"
+            fileAssociation(
+                    "text/kotlin",
+                    "kot",
+                    "Kotlin Source File0",
+                    project.file("Kotlin_icon_big.png"),
+                    project.file("Kotlin_icon_big.ico"),
```

Comment:
I checked it on Windows, and it doesn't seem to work:
<img width="470" alt="image" src="https://github.com/JetBrains/compose-multiplatform/assets/5963351/b9801494-b7ec-42e5-861f-6bddc12acb77">


It has the app icon instead.

JDK: `openjdk 19.0.2 2023-01-17`

Reply:
Never mind. It works.

I put icons in the root folder instead of the module folder

Reply:
Can you give the example standalone?

Reply:
Do you need if it works? :) I was mistaken.

Reply:
I just had out-of-date message history, no need, of cource.